lib/test.gi

```diff
@@ -124,6 +124,8 @@ InstallGlobalFunction(ParseTestInput, function(str, ignorecomments, fnam)
       Append(inp[Length(inp)], lines[i]{[3..Length(lines[i])]});
       Add(inp[Length(inp)], '\n');
       i := i+1;
+    elif Length(lines[i]) > 1 and lines[i]{[1..2]} = ">\t" then
+        testError("Invalid test file: Continuation '> ' uses a tab instead of a space");
```

Comment:
When reading error messages like this, I am always confused whether they tell me what *is*, or what *should be*. Perhaps this could be made clearer? How about this?
```suggestion
        testError("Invalid test file: Continuation prompt '> ' followed by a tab, expected a regular space");
```